frappe/database/__init__.py

```diff
@@ -39,7 +39,7 @@ def drop_user_and_database(db_name, root_login=None, root_password=None):
 		)
 
 
-def get_db(host=None, user=None, password=None, port=None):
+def get_db(host, port, user, password):
```

Comment:
Don't change kwargs to args and arg order unless there's any real need for it. This breaks old code.

Reply:
Indeed that was just a good way of validating the interface (during refactoring).